Src/GeneratedTests/Discovery.Tests/resources/urlshortener.json

```diff
@@ -0,0 +1,319 @@
+﻿{
```

Comment:
I'd rather keep this as the real response. I agree the risk is small, but removing anything here could remove something that triggers an error/bug.
The only downside with keeping everything is the size of this resource, but it's not enormous, so I don't particularly see a problem.